satellite/contact/endpoint.go

```diff
@@ -42,23 +49,23 @@ func (endpoint *Endpoint) CheckIn(ctx context.Context, req *pb.CheckInRequest) (
 
 	peerID, err := identity.PeerIdentityFromContext(ctx)
 	if err != nil {
-		return nil, rpcstatus.Error(rpcstatus.Internal, Error.Wrap(err).Error())
+		return nil, rpcstatus.Error(rpcstatus.Unknown, errCheckInIdentity.New("failed to get ID from context: %v", err).Error())
```

Comment:
Can we put corresponding log lines at each of the return points? Or do something else to make returned errors always be logged with some request details

Reply:
sounds good, let me add some log lines